qt/aqt/importing.py

```diff
@@ -460,39 +451,11 @@ def _replaceWithApkg(mw: aqt.AnkiQt, filename: str, backup: bool) -> None:
     mw.progress.start(immediate=True)
 
     def do_import() -> None:
-        z = zipfile.ZipFile(filename)
-
-        # v2 scheduler?
-        colname = "collection.anki21"
-        try:
-            z.getinfo(colname)
-        except KeyError:
-            colname = "collection.anki2"
-
-        with z.open(colname) as source, open(mw.pm.collectionPath(), "wb") as target:
-            # ignore appears related to https://github.com/python/typeshed/issues/4349
-            # see if can turn off once issue fix is merged in
-            shutil.copyfileobj(source, target)
-
-        d = os.path.join(mw.pm.profileFolder(), "collection.media")
-        for n, (cStr, file) in enumerate(
-            json.loads(z.read("media").decode("utf8")).items()
-        ):
-            mw.taskman.run_on_main(
-                lambda n=n: mw.progress.update(  # type: ignore
-                    tr.importing_processed_media_file(count=n)
-                )
```

Comment:
This is potentially quite slow when users have a lot of media, so we probably need to provide progress info. check_media_folder() and checkmedia.py is an example of how we can do this.